test/legacy_test/test_ops_nms.py

```diff
@@ -203,7 +208,7 @@ def fun(x):
                         0.1,
                         paddle.to_tensor(scores),
                         paddle.to_tensor(category_idxs),
-                        categories,
+                        paddle.to_tensor(categories),
```

Comment:
如果当前 pr 不需要适配动转静，那么麻烦复原一下这行更改吧
```suggestion
                        categories,
```

Reply:
这个在修改完 nms 中的 `cur_category_boxes_idxs = paddle.where(paddle.equal(category_idxs, category_id))[0]` ,不这么改本地测试会报错

Reply:
![image](https://github.com/PaddlePaddle/Paddle/assets/106524776/d9e44bc1-0bca-4ba1-b739-1e949c86aa91)

Reply:
> 这个在修改完 nms 中的 `cur_category_boxes_idxs = paddle.where(paddle.equal(category_idxs, category_id))[0]` ,不这么改本地测试会报错

好的了解了。那就保留更改吧